nui-libgdx/src/main/java/org/terasology/nui/backends/libgdx/LibGDXMouseDevice.java

```diff
@@ -86,4 +110,9 @@ public boolean isVisible() {
     public void setGrabbed(boolean grabbed) {
         Gdx.input.setCursorCatched(grabbed);
     }
+
+    @Override
+    public int getMaxPointers() {
+        return 20;
```

Comment:
This is the same magic number as the array length - worth a constant in this class?